src/lib.rs

```diff
@@ -1954,6 +1954,13 @@ impl Url {
 
         parser.serialization.push_str(self.slice(old_scheme_end..));
         self.serialization = parser.serialization;
+
+        // Update the port so it ca be removed
```

Comment:
can* :)

Reply:
Oh right, I'll blame the keyboard for this nit :p 
Fixing it as soon as I get back home !